ehitchhiking-ui/src/app/shared/components/rate-passangers-modal/rate-passangers-modal.component.ts

```diff
@@ -0,0 +1,33 @@
+import {Component} from '@angular/core';
+import {MatDialogRef} from '@angular/material';
+import {Passanger} from './passanger';
+
+@Component({
+  selector: 'rate-passangers-modal',
+  templateUrl: './rate-passangers-modal.component.html',
+  styleUrls: ['./rate-passangers-modal.component.sass']
+})
+export class RatePassangersModalComponent {
+  ratingClicked: number;
```

Comment:
Why do you need this variable? There in no reason to save last selected rating value